marlowe-playground-client/src/Home.purs

```diff
@@ -3,42 +3,87 @@ module Home where
-        , div [ classes [ ClassName "rectangle" ] ] [ img [ src option3 ] ]
+    , div
+        [ classNames [ "flex", "mb-8" ] ]
+        [ div
+            [ classNames (newProjectClasses <> [ "mr-24" ])
+            , onClick (const <<< Just $ NewProjectAction $ NewProject.CreateProject Javascript)
+            ]
+            [ img [ src newProjectJavascriptIcon, classNames [ "h-16", "mb-4" ] ]
+            , text
+                "Start in Javascript"
+            ]
+        , div
+            [ classNames (newProjectClasses <> [ "mr-24" ])
+            , onClick (const <<< Just $ NewProjectAction $ NewProject.CreateProject Haskell)
+            ]
+            [ img [ src newProjectHaskellIcon, classNames [ "h-16", "mb-4" ] ]
+            , text
+                "Start in Haskell"
+            ]
+        , div [ classNames [ "border-0", "border-l", "border-black", "border-solid", "h-10", "mt-2" ] ] []
+        , div
+            [ classNames (newProjectClasses <> [ "ml-24", "mr-4" ])
+            , onClick (const <<< Just $ NewProjectAction $ NewProject.CreateProject Marlowe)
+            ]
+            [ img [ src marloweLogo, classNames [ "h-16", "mb-4" ] ]
+            , text
+                "Start in Marlowe"
+            ]
+        , div [ classNames [ "flex", "flex-col" ] ]
+            [ img [ classNames [ "mt-4" ], src arrowRightUp ]
+            , img [ classNames [ "mt-3" ], src arrowLeftUp ]
+            ]
+        , div
+            [ classNames (newProjectClasses <> [ "ml-4", "mr-1" ])
+            , onClick (const <<< Just $ NewProjectAction $ NewProject.CreateProject Blockly)
+            ]
+            [ img [ src newProjectBlocklyIcon, classNames [ "h-16", "mb-4" ] ]
+            , text
+                "Start in Blockly"
+            ]
         ]
-    , div [ classes [ ClassName "ready-to-go-buttons" ] ]
-        [ h2_ [ text "Ready to go?" ]
-        , button [ onClick ((const <<< Just <<< OpenModal) NewProject) ] [ text "Start a new project" ]
-        , div [ classes [ ClassName "links" ] ]
-            [ a [ href "./doc/marlowe/tutorials/index.html", target "_blank" ] [ text """Read our "Getting Started" guide""" ]
-            , vl
-            , a [ onClick ((const <<< Just <<< OpenModal) OpenDemo) ] [ text "Browse the example files" ]
+    , div [ classNames [ "mb-10", "flex", "items-end" ] ]
+        [ img [ classNames [ "mr-24" ], src arrowRightDown ]
+        , div [ classNames [ "flex", "flex-col", "text-sm" ] ]
+            [ img [ src simulationIconBlack, classNames [ "mb-4" ] ]
+            , text "Simulate"
             ]
+        , img [ classNames [ "ml-24" ], src arrowLeftDown ]
+        ]
+    , div [ classNames [ "font-bold", "text-sm" ] ]
+        [ a [ href "./doc/marlowe/tutorials/index.html", target "_blank" ] [ text "Read our Getting Started guide" ]
         ]
     ]
+  where
+  newProjectClasses = [ "flex", "flex-col", "font-bold", "text-sm", "cursor-pointer" ]
```

Comment:
add `"text-center"` (for when narrower screens cause a line wrap)

Reply:
done! but in any case, this website does not have a mobile version :/